StripeConnect/StripeConnect/Source/Internal/Extensions/UIViewController+StripeConnect.swift

```diff
@@ -0,0 +1,19 @@
+//
+//  UIViewController+StripeConnect.swift
+//  StripeConnect
+//
+//  Created by Mel Ludowise on 10/11/24.
+//
+
+import UIKit
+
+extension UIViewController {
+    /// Helper that adds a child view controller and pins its view to this view controller's view
+    func addChildAndPinView(_ child: UIViewController) {
+        child.willMove(toParent: self)
+        addChild(child)
+        view.addSubview(child.view)
+        child.view.frame = view.bounds
```

Comment:
Shouldn't this use constraints?

Reply:
We don't need to if `translatesAutoresizingMaskIntoConstraints = true`, which it is by default